.github/workflows/tests.yml

```diff
@@ -1,22 +1,30 @@
 name: Tests
-on: [push]
-jobs:
+on:
+  push:
+    branches:
+      - main
+  pull_request:
 
-  tests-on:
-    name: ${{matrix.go-version}} ${{matrix.os}}
+jobs:
+  tests-off:
+    name: ${{ matrix.os }} - Go v${{ matrix.go-version }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        go-version: [1.12.x, 1.13.x]
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        go-version:
+          - "1.16.x"
```

Comment:
We can drop support for 1.16 imho, makes working with `embed` easier and traditionally gobuffalo would always support only the last 2 go versions

Reply:
Yeah, I think so finally. I just leave it as is for during we officially support 1.17 and 1.18, then we can remove that.

Reply:
obsoleted